Casks/p4v.rb

```diff
@@ -10,10 +10,12 @@
   homepage "https://www.perforce.com/products/helix-core-apps/helix-visual-client-p4v"
 
   livecheck do
-    url "https://www.perforce.com/perforce/doc.current/user/p4vnotes.txt"
+    url "https://www.perforce.com/support/software-release-index#tab-panel-6258"
```

Comment:
```suggestion
    url "https://www.perforce.com/support/software-release-index"
```

The fragment (`#tab-panel-6258`) doesn't affect the content of the page, so it may be best to omit it. Eventually I'll get around to finishing up a caching setup for livecheck (I've already merged most of the groundwork) but omitting the fragment here would be helpful for identifying duplicate URLs in the future (e.g., if this page was also checked in another formula/cask in the same run).